public/pug/register.pug

```diff
@@ -0,0 +1,17 @@
+extends layout
+
+block styles
+    link(rel="stylesheet" type="text/css" href="../css/login.css")
+
+block main
+    #boarder
+        h1 register
+        #credentials
+            p=messages.error
+            form(action="api/register", method="post")
+                .textbox
+                    input(type='username', id="username", name="username" placeholder='Username' required)
+                .textbox
+                    input(type='password', id="password", name="password" placeholder='Password' required)
+                button.blue-btn(type="submit") register
+                a.black-link#spaced(href="login") sign in
```

Comment:
This whole block (and `/login`) could be refactored as a [mixin](https://pugjs.org/language/mixins.html) to be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)